tests/macros/DefaultAPISidebar.test.js

```diff
@@ -0,0 +1,359 @@
+/**
+ * @prettier
+ */
+const { JSDOM } = require('jsdom');
+
+const {
+    beforeEachMacro,
+    describeMacro,
+    itMacro,
+    lintHTML
+} = require('./utils');
+
+/**
+* Load all the fixtures.
+*/
+const fs = require('fs');
+const path = require('path');
+const subpagesFixturePath = path.resolve(__dirname, 'fixtures/defaultapisidebar/subpages.json');
+const subpagesFixture = JSON.parse(fs.readFileSync(subpagesFixturePath, 'utf8'));
```

Comment:
I think `subpagesJSON` rather than `subpagesFixture` would be a clearer name and more consistent with your naming scheme?